packages/office-ui-fabric-react/src/components/Nav/Nav.styles.ts

```diff
@@ -173,24 +172,24 @@ export const getStyles = (props: INavStyleProps): INavStyles => {
           },
           '&:hover': {
             color: semanticColors.bodyText,
-            backgroundColor: palette.neutralLighterAlt
+            backgroundColor: semanticColors.bodyBackgroundHovered
           },
           [`.${classNames.compositeLink}:hover &`]: {
             color: semanticColors.bodyText,
-            backgroundColor: palette.neutralLighterAlt
+            backgroundColor: semanticColors.bodyBackgroundHovered
           }
         }
       },
       isGroup && {
         fontSize: fonts.large.fontSize,
         width: '100%',
-        height: `${navHeight}px`,
+        height: navHeight,
         borderBottom: `1px solid ${semanticColors.bodyDivider}`
       },
       isLink && {
         display: 'block',
         width: `${leftPaddingExpanded - 2}px`,
```

Comment:
weird that this isn't `leftPaddingExpanded - 2`. maybe fix while you're there. :)

Reply:
also curious why -2. is there a mixing `box-sizing` prop?

Reply:
the "top: 1px" seems to indicate this was on purpose to float it with 1px padding